build/msvc.rs

```diff
@@ -51,27 +44,28 @@ pub fn build_and_link() -> IncludePaths {
         .warnings(false)
         .compile("libffi");
 
-    IncludePaths(INCLUDE_DIRS.iter().map(Into::into).collect())
+    //IncludePaths(INCLUDE_DIRS.iter().map(Into::into).collect())
+}
+
+pub fn probe_and_link() {
+    // TODO
```

Comment:
@timfish Does MSVC support dynamic linking of libffi? If so, how do we implement that here as I'm not sure the pkg-config crate works on Windows. If not, then I think we can just make this function an alias of `build_and_link`.

Reply:
Sorry no idea, I've never tried!

Reply:
@timfish No worries. I'll change the code to just call `build_and_link()`, that way we don't break any MSVC setups. We can always change this in the future if necessary.